pkg/didcomm/protocol/introduce/service.go

```diff
@@ -95,6 +100,9 @@ type Service struct {
 type Provider interface {
 	OutboundDispatcher() dispatcher.Outbound
 	StorageProvider() storage.Provider
+	// this method should be provided by the introduce client
+	// method should be implemented in didexchange Client
+	SendInvitation(inv *didexchange.Invitation, dest *service.Destination) error
```

Comment:
@llorllale @troyronda @rolsonquadras  I am going to get it directly from the context, Are you ok with that?
See https://github.com/hyperledger/aries-framework-go/pull/805#discussion_r349628787

```
svc, err := ctx.Service(didexchange.DIDExchange)
```

Reply:
@soluchok  DIDExchange service doesn't send invitations. Invitations are sent through out-of band transports. Is this change as part of [introduce#response](https://github.com/hyperledger/aries-rfcs/tree/master/features/0028-introduce#response) requirement ?

Reply:
@rolsonquadras  `the final step in the protocol is to begin the DID Exchange protocol by forwarding the invitation message from one introducee to the other.` We do not have `SendInvitation` function in DIDExchange yet, we need to add it.

Reply:
yes